thirdeye/thirdeye-frontend/app/styles/wrapper/styles.scss

```diff
@@ -159,6 +159,182 @@ body {
   }
 }
 
+.te-alert-page-top {
+  background: white;
+  width: 100%;
+  border-top: 1px app-shade(black, 1) solid;
+}
+
+.te-alert-page-selectors {
+  &__range-picker {
+    width: 23.5%;
+  }
+
+  &__label {
+    margin: 15px 0 5px 0;
+    padding: 0 0 2px 3px;
+    font-weight: 500;
+    font-size: 16px;
+    color: app-shade(black, 9);
+  }
+
+  &__side-link {
+    font-size: 16px;
+    color: #0073b1;
+    font-weight: 600;
+    letter-spacing: .03em;
+    float: right;
+    margin-top: -15px;
+
+    &--high {
+      margin-top: -30px;
+    }
+  }
+}
+
+.te-alert-page-cards {
+  @include group-clearfix;
+  font-family: $source-sans-pro;
+  margin-top: 20px;
+
+  &__card {
+    list-style: none;
+    float: left;
```

Comment:
using display flex might be easier here: https://codepen.io/anon/pen/BmNBXx

Reply:
nice - thanks.